content/en/_index.html

```diff
@@ -13,30 +13,322 @@
+            </li>
+            <li>
+              Portable and immutable, you can copy an artifact with its
+              signature across registries
+            </li>
+          </ul>
+        </div>
+      </div>
+    </div>
+    <div class="feature-card">
+      <div class="order-number is-digit">05</div>
+      <div class="feature-title">
+        <h1 class="text-left">
+          Community- <br />
+          driven
+        </h1>
+      </div>
+      <div class="feature-description text-left">
+        <div class="description is-bullets">
+          <ul>
+            <li>
+              100% open source, built and improved by the active communitye
+            </li>
+            <li>100+ contributors in total, from multiple organizations</li>
+            <li>Fast iteration cadence and open community governance</li>
+          </ul>
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+{{% /blocks/lead %}} {{% blocks/lead color="white" %}}
+<h1 class="title">Adopted and trusted by</h1>
+<p class="width-60 is-centered font-size-md">
+  Industry-leading enterprises and organizations are using Notary v2 for
+  research, production, and integration with security products. If you are using
+  Notary v2, please share your case with us
+</p>
 
-  <ul>
-    <li>Sign, store, and verify any OCI artifact, including container images</li>
-    <li>Signing and verifying uses public/private key verification</li>
-    <li>Signing does not change the <code>@digest</code> or <code>artifact:tag</code> reference</li>
-    <li>Signatures are portable across OCI-conformant registries</li>
-    <li>Support for multiple signatures</li>
-    <li>Designed to support multiple development flows</li>
-  </ul>
+<div
+  class="flex gap-md width-80 is-centered font-size-sm margin-md-top is-mobile-responsive fullwidth-tablet"
+>
+  <div class="horizontalCard flex gap-md is-align-items-center">
+    <div class="img-container-md">
+      <img src="aws_logo.svg" alt="Aqua logo" />
+    </div>
+    <div class="text-container text-left">
+      <p>
+        AWS Signer team is using and contributing to Notation, building the
+        cryptographic signing services for customers
+      </p>
+    </div>
+  </div>
+  <div class="horizontalCard flex gap-md space-between is-align-items-center">
+    <div class="img-container-md">
+      <img src="microsoft_logo.svg" alt="Aqua logo" />
+    </div>
+    <div class="text-container text-left">
+      <p>
+        Notation is widely adopted by multiple Microsoft teams and services,
+        such as Windows container team, AKS team, Azure Code Signing service,
+        Ratify, etc.
+      </p>
+    </div>
+  </div>
+  <div class="horizontalCard flex gap-md space-between is-align-items-center box">
+    <div class="img-container-md">
+      <img src="zot.svg" alt="Zot logo" />
+    </div>
+    <div class="text-container text-left">
+      <p>
+        Zot registry supports store Notation signature as OCI artifacts
```

Comment:
Do we have confirmation from the Zot team that we can use their name?

Reply:
Yes. We confirmed with Ramkumar from the Zot team in a community call.